### DISCUSSION TITLE
New ideas discussion

### DISCUSSION THREAD

**Discussion by CorpHackRyan** [CorpHackRyan/RP-Spaceoids#20] [Ideas]
I don't know whether to put new idea submissions as an issue, or make a discussion about them. that's what we're here to figure out! Also, I'm sticking this in the Ideas category. Since it won't let you post a new discussion without selecting which category it belongs to. 

anyway, I was glancing over my computer game controller that I've never used not once and thought that might be a good idea.. 

.. add the ability to use a game controller as input

**Comment by patpragman** [comment]
I don't think this would be crazy.  Pygame had a module for that I think?
